app/Http/Middleware/AssetCountForSidebar.php

```diff
@@ -52,6 +52,13 @@ public function handle($request, Closure $next)
             \Log::debug($e);
         }
 
+        try {
+            $total_byod_sidebar = Asset::where('byod', '=', '1')->count();
+            view()->share('total_byod_sidebar', $total_byod_sidebar);
+        } catch (\Exception $e) {
+            \Log::debug($e);
```

Comment:
This feels a little much - if we have an exception in this condition, I feel like we should have an actual exception?